client/src/components/Kanban/Board.tsx

```diff
@@ -1,17 +1,19 @@
 import { Grid } from '@material-ui/core';
 import { DragDropContext } from 'react-beautiful-dnd';
-import { useKanban } from '../../context/useKanbanContext';
+import { IBoard } from '../../interface/Board';
+import { useKanban } from '../../context/';
 import { FocusCard } from './FocusCard/FocusCard';
 import Column from './Column/Column';
 
-const Board = (): JSX.Element => {
-  const { columns, handleDragEnd } = useKanban();
+const Board = ({ activeBoard }: { activeBoard: IBoard }): JSX.Element => {
+  const { handleDragEnd } = useKanban();
+
   return (
     <DragDropContext onDragEnd={handleDragEnd}>
       <FocusCard />
       <Grid container>
-        {columns.map((column) => (
-          <Column key={column.id} {...column} />
+        {activeBoard.columns.map((column) => (
+          <Column key={column._id} id={column._id} name={column.name} cards={column.cards} />
         ))}
```

Comment:
Mapping over the `activeBoard`'s columns instead of the columns setup in `KanbanContext`